x-pack/plugins/cloud_security_posture/common/types/rules/v4.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import { BenchmarksCisId } from '../latest';
+
+export type {
+  cspBenchmarkRuleMetadataSchema,
+  CspBenchmarkRuleMetadata,
+  cspBenchmarkRuleSchema,
+  CspBenchmarkRule,
+  FindCspBenchmarkRuleResponse,
+  CspSettings,
+  CspBenchmarkRulesStates,
+} from './v3';
+
+const DEFAULT_BENCHMARK_RULES_PER_PAGE = 25;
+
+export const findCspBenchmarkRuleRequestSchema = schema.object({
```

Comment:
@animehart was there any breaking change to the interfaces so we needed to introduce a new version?